component/site/libraries/helper.php

```diff
@@ -2060,6 +2060,10 @@ function isAdminUser($user = null)
 			$user = JFactory::getUser();
 		}
 		//$access = JAccess::check($user->id, "core.admin","com_jevents");
+		// Add a second check incase the getuser failed.
+		if (!$user) {
```

Comment:
It can fail when a userid isn't passed into the call. So this avoids a fatal error. Happens when a userid exists but the user has been deleted from Joomla!